src/main/drivers/accgyro/accgyro_mpu.c

```diff
@@ -359,23 +359,25 @@ void mpuGyroInit(gyroDev_t *gyro)
 
 uint8_t mpuGyroDLPF(gyroDev_t *gyro)
 {
-    uint8_t ret;
-    if (gyro->gyroRateKHz > GYRO_RATE_8_kHz) {
-        ret = 0;  // If gyro is in 32KHz mode then the DLPF bits aren't used - set to 0
-    } else {
+    uint8_t ret = 0;
+    
+    // If gyro is in 32KHz mode then the DLPF bits aren't used
+    if (gyro->gyroRateKHz <= GYRO_RATE_8_kHz) {
         switch (gyro->hardware_lpf) {
             case GYRO_HARDWARE_LPF_NORMAL:
```

Comment:
We should add the default case here.

Reply:
The default is 0 and the `ret` variable is initialized.  I actually removed the `default` clause because it wasn't necessary.

Reply:
To me, having an explicit default makes it explicit what should be the (for the value being tested), and it makes the code more robust to change.